back/spec/models/initiative_spec.rb

```diff
@@ -115,6 +115,26 @@
     end
   end
 
+  describe '#proposed_at' do
+    before do
+      allow(Time).to receive(:now).and_return(Time.now)
```

Comment:
Why are you stubbing `Time`?

Reply:
No idea. Probably a useless copy/paste. Removed